redis.go

```diff
@@ -328,6 +328,7 @@ func (c *baseClient) _process(ctx context.Context, cmd Cmder) error {
 			retryTimeout := true
 			err := c.withConn(ctx, func(ctx context.Context, cn *pool.Conn) error {
 				err := cn.WithWriter(ctx, c.opt.WriteTimeout, func(wr *proto.Writer) error {
+					internal.Instruments.WriteCount.Add(ctx, 1)
```

Comment:
Perhaps it makes sense to move this to internal/pool/conn.go and instrument `WithWriter`

Reply:
Moved it to within `WithWriter` where the `WriteCount` will be incremented if `WithWriter` runs until completion. Please let me know if you think this is correct.